### ISSUE TITLE
[style]: Background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by amelia2802** [SurajPratap10/Imagine_AI#259]
### What's the style idea?

We need to improve the background and foreground colors as it is difficult for many users to read.

### Add screenshots

Add screenshots[Accessibility Report]
![Accessibility Report](https://github.com/SurajPratap10/Imagine_AI/assets/49182604/9cc273ca-a6ab-4293-980c-8ecf4e340677)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by amelia2802** [comment]
Hi @SurajPratap10 , I have raised this issue under gssoc'23.
 It'd be great if you review this and assign me.

**Comment by SurajPratap10** [comment]
Assigned @amelia2802

**Comment by amelia2802** [comment]
Thank you @SurajPratap10 for assigning, before I start working on this issue do you have any particular brand color dedicated to Imagine_AI? Or I can use any contrasts?

**Comment by SurajPratap10** [comment]
No, nothing as such. You can innovate @amelia2802  your style. If there would be any enhancements then I will tell that in the PR itself

**Comment by amelia2802** [comment]
Alright!
